<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeywordsPageForm</class>
 <widget class="QWidget" name="KeywordsPageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Keywords</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="keywordLabel">
              <property name="font">
               <font>
                <family>MS Sans Serif</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Keyword: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="keywordLineEdit">
              <property name="font">
               <font>
                <family>MS Sans Serif</family>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="existingKeywordsButton">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Existing Keywords</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QListWidget" name="listWidget"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="insertKeywordButton">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Insert Keyword</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="removeKeywordButton">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Remove Keyword</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
